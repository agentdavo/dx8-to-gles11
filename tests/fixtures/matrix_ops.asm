mload 1.0, 2.0, 3.0
loadi
