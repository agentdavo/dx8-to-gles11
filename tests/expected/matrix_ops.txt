MATRIX_LOAD 1.0 2.0 3.0 1.0
LOAD_IDENTITY
